content/maintenance-guidelines.md

```diff
@@ -0,0 +1,119 @@
+# Open source maintenance guidelines
+
```

Comment:
Maybe we can give a short foreword here. Just a couple sentences of context for folks who are linked directly into this page.